packages/@azure/servicebus/data-plane/lib/subscriptionClient.ts

```diff
@@ -328,47 +324,31 @@ export class SubscriptionClient extends Client {
 
   /**
    * Lists the sessions on the ServiceBus Subscription.
-   * @param skip The number of sessions to skip
-   * @param top Maximum numer of sessions.
+   * @param maxSessionCount Maximum number of sessions.
```

Comment:
minor: I would name it just 'max' or 'maxNumberOfSessions'
maxSessionCount could mean max messages count in a session (possibly, at first glance-ish)